app/helpers/application_helper.rb

```diff
@@ -1285,6 +1284,11 @@ def accessible_select_event_types
     ApplicationController::Timelines::SELECT_EVENT_TYPE.map { |key, value| [_(key), value] }
   end
 
+  def storage_systems_options
+    storages = PhysicalStorage.all.ids
+    storages.map{|k| [PhysicalStorage.find_by(:id=>k).name, k]}
```

Comment:
Hm there is a lot wrong with this...

1. I don't think this should be in the base application_helper when it is storage_system specific
2. It queries for physical storages one-at-a-time rather than in batches
3. It is showing ALL physical storages regardless of tenant/group/RBAC

This needs a re-think IMO

Reply:
I modified the function so it will be less problematic, please tell me if this is fine with you:
<img width="313" alt="Screen Shot 2022-09-12 at 10 23 53" src="https://user-images.githubusercontent.com/62609377/189596007-c4cb0df8-a858-43ca-9193-bfcda83a40bd.png">

Also, where should I place it? 
The timeline JS files are in the UI repo in this is relevant to it

Reply:
If you want to return an array of `[name, id]` the way to do that in the most efficient way possible is `PhysicalStorage.pluck(:name, :id)`

I still don't think we can return all physical storages here though, if you have two tenants wouldn't tenant A see all physical storages in tenant B and vice versa?

Reply:
Changed it to the pluck method you suggested, and it looks good.
For the other part, I can use the Autosde::PhysicalStorage, is this better? it will make sure only AutoSDE objects are returned

Reply:
I would hope they are already reviewing this, but /cc @jeffibm @DavidResende0 @GilbertCherrie